components/Tactics/TacticsEdit/StyledTacticsEdit.js

```diff
@@ -47,7 +47,7 @@ export function TacticsSubmitButton({ children }) {
         <Spinner />
       ) : (
         <>
-          <SubmitIcon disabled="false" /> {children}
+          <SubmitIcon disabled={false} /> {children}
```

Comment:
I don't see a reason to use `disabled={false}` here because you do not change it at any time.

Reply:
Same as for EditIcon 😊